core/jobs/batch_jobs/skill_migration_jobs.py

```diff
@@ -243,24 +244,30 @@ def run(self) -> beam.PCollection[job_run_result.JobRunResult]:
                 lambda skill_summary_model: skill_summary_model.id)
         )
 
-        migrated_skill_results = (
+        all_migrated_skill_results = (
             unmigrated_skill_models
             | 'Transform and migrate model' >> beam.MapTuple(
                 self._migrate_skill)
         )
-        migrated_skills = (
-            migrated_skill_results
-            | 'Filter oks' >> beam.Filter(
-                lambda result_item: result_item.is_ok())
-            | 'Unwrap ok' >> beam.Map(
-                lambda result_item: result_item.unwrap())
-        )
+
         migrated_skill_job_run_results = (
-            migrated_skill_results
+            all_migrated_skill_results
             | 'Generate results for migration' >> (
                 job_result_transforms.ResultsToJobRunResults('SKILL PROCESSED'))
         )
 
+        filtered_migrated_skills = (
+            all_migrated_skill_results
+            | 'Filter migration results' >> (
+                filter_results_transforms.FilterResults())
```

Comment:
You can now remove the `AuditSkillMigrationJob`

Reply:
ditto for story

Reply:
Done